loader/loader.c

```diff
@@ -2850,6 +2888,43 @@ static VkResult loaderReadLayerJson(const struct loader_instance *inst, struct l
         goto out;
     }
 
+    props->num_blacklist_layers = 0;
+    props->blacklist_layer_names = NULL;
+    blacklisted_layers = cJSON_GetObjectItem(layer_node, "blacklisted_layers");
+    if (blacklisted_layers != NULL && strcmp(name, VK_OVERRIDE_LAYER_NAME)) {
+        loader_log(inst, VK_DEBUG_REPORT_WARNING_BIT_EXT, 0,
+                   "Layer %s contains a blacklist, but a blacklist can only be provided by the override metalayer. "
+                   "This blacklist will be ignored.",
+                   name);
```

Comment:
Won't this spit out for every layer added (like via Environment variables) in addition to the override layer instead of just once?